### ISSUE TITLE
Panel Edit: Edit Panel-B causes Panel-A to get changed also

### ISSUE THREAD

**Issue by bob454522** [grafana/grafana#98351]
### What happened?

in grafana v11.4.0  (and also 11.3.x ) - if you duplicate a panel,  then any edits you make to the duplicated panel get applied to the original panel as well.   the only recent grafana version that does NOT have this bug is v11.2.0.

see screen cap:
![Image](https://github.com/user-attachments/assets/b60c9edc-aa1d-4e3d-840e-5d1da586a108)

### What did you expect to happen?

only change the panel i was editing

### Did this work before?

yes in v11.2.0 and prior

### How do we reproduce it?

1. dup a panel
2. make a edit to dup and save
3. reload / change time frame

### Environment (with versions)?

Grafana: 11.4.0
OS: ubuntu
Browser: FF or Chrome

### Grafana platform?

A downloaded binary

### Datasource(s)?

any

**Comment by bob454522** [comment]
i can also confirm that this effects both grafana and grafana-enterprise versions (as expected).

is still an issue as of versions 11.3.x  through    grafana-dev:11.5.0-214835   (and ofcouse also: grafana-enterprise-dev:11.5.0-80518).

this is a pretty big bug, as it makes it very (very) difficult to edit any dashboards with several panels.

**Comment by bob454522** [comment]
sample of the bug on Play.grafana.org
(I just don't understand how im the only one finding this after this many weeks now, as duplicating panels is such a common use case);

![Image](https://github.com/user-attachments/assets/03675cd8-7ea1-434f-9d81-dd9445072e2d)

**Comment by leeoniya** [comment]
previous report: https://github.com/grafana/grafana/issues/97223

> sample of the bug on Play.grafana.org

this is helpful, i can see the big on play.

- i was not able to reproduce this locally with our `grafana-testdata` data source
- i was not able to reproduce it with Scenes disabled (by adding `&scenes=false` to the URL)
- i was not able to get edits to other panel options (like field overrides or viz options) to duplicate

this may be a bug with a combo of the Graphite query editor and the Scenes architecture. i see this was also reported in the prior issue for the Google Cloud Monitoring data source, so it may be specific to panel duplication, query editors, and Scenes.

**Comment by leeoniya** [comment]
there's an open PR that should fix this at least for Graphite: https://github.com/grafana/scenes/pull/967

**Comment by fyang13** [comment]
This bug makes working on dashboard panels very unproductive, any this be given a higher priority?

**Comment by bob454522** [comment]
> This bug makes working on dashboard panels very unproductive, can this be given a higher priority?

I totally agree, it was a big issue, however, for me, it seems to have been resolved with one of the updates (i'm currently running 11.5.0 and not having the issue any longer)  - please reply back with your grafana server version, I really hope the bug hasn't crept back in to versions after 11.5.0
tks

**Comment by oscarkilhed** [comment]
This should be fixed with https://github.com/grafana/scenes/pull/967 that will be in upcoming releases.
